app/generic/verifier/facades/acapy_verifier_utils.py

```diff
@@ -23,7 +23,10 @@ def record_to_model(
             connection_id=record.connection_id,
             created_at=record.created_at,
             protocol_version=ProofRequestProtocolVersion.v20.value,
-            presentation=record.pres,
+            presentation=record.by_format.pres["indy"]
+            if record.by_format.pres
+            else record.by_format.pres,
```

Comment:
`record.by_format.pres` will always be `None` right? In that case we can maybe use `None` to make it more explicit?

```suggestion
            else None
```

Reply:
fair point